### PR TITLE
Remove status from Details card, add uptime to Overview

### LINKED ISSUE #893: VM Details - remove Status field from Details Card and move uptime to Overview Card
screenshot-localhost-3000-2018 12 13-12-31-36

~~On the details card, the **Status** field shows the VM's status and the VM uptime.  Currently the formatted uptime follows the output of the unix `uptime` command.  A more user friendly formatting is preferred.~~

The Details Card duplicates the **Status** field from the Overview Card.  The field should be removed and the uptime should be added to Overview's **Status** field.  A more user friendly formatting is preferred (as discussed in the comments below).

For reference, the formatting is done by the [`moment-duration-format` package](https://github.com/jsmreese/moment-duration-format).  Currently the format is (`src/utils/format.js`):

````js
  const formatted = moment.duration(durationInMS).format('d [days], h:mm')
````

### PR DESCRIPTION
**Pull request by sjd78**
- removed the status and uptime fields from the details card

 - added uptime to the overview card

 - updated the uptime format as per the issue

Resolves #893

### CODE REVIEW

File: src/utils/format.js [hunk 1]
```diff
@@ -26,6 +26,9 @@ export function formatUptimeDuration (
     return defaultValue
   }
 
-  const formatted: string = moment.duration(durationInMS).format('d [days], h:mm')
+  // (up for 0 days, 0 hours, 1 minute, 37 seconds)
+  const formatted: string = moment.duration(durationInMS).format(
```
Comment (bond95) [inline src/utils/format.js]: I think it is better to shut off seconds after 2 minutes as @gregsheremeta mentioned in issue https://github.com/oVirt/ovirt-web-ui/issues/893#issuecomment-447343729

Comment (sjd78) [reply src/utils/format.js]: Fixed

**Comment by leistnerova** [comment]
ci build please

**Comment by leistnerova** [comment]
Status only once in overview and correctly formatted. But it shows wrong uptime when VM was suspended, it doesn't count the pause and starts from 0 again (admin portal shows in VM grid correct uptime).

**Comment by sjd78** [comment]
@leistnerova - So looking at the webadmin code for the VM table, uptime column, and it uses "elapsed time" from the VM statistics object.  That field is not currently available for VM statistics via the REST API.  Easy enough to add there, so I'm doing that.

I'll tweak the REST API to pass back the extra statistic and then use that.

While that patch is being reviewed etc, I'll update this PR to look for it and fall back to the VM start time if the statistic isn't there.  The difference being that **elapsed time** doesn't include VM suspend time while **startTime** does (I assume a suspended VM is still considered "started").

**Comment by sjd78** [comment]
(rebased)

**Comment by sjd78** [comment]
The uptime string will now use the **elapsedUptime** field from VmStatistics, if it is available, and **startTime** from the VM if it is not.  Checks on the Details card that require the VM status field have also been fixed (by adding back the status const).

Note: **elapsed.time** was added to the REST API for VM statistics via https://gerrit.ovirt.org/#/c/96683/ and is transformed to **statistics.elapsedUptime**.

**Comment by leistnerova** [comment]
ci build please

**Comment by leistnerova** [comment]
LGTM

**Pull request merged**